.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+    build:
+
```

Comment:
Do we need these empty lines? I guess no.

Reply:
Removed